PyMca5/tests/ut/test_tiled_catalog_selector.py

```diff
@@ -25,68 +25,61 @@ def test_init(optional_args: Mapping[str, Any]):
     TiledCatalogSelector()
```

Comment:
@padraic-shafer should the `optional_args` be passed into the `TiledCatalogSelector` in the `test_init` test? Looks like they aren't being used right now.

Reply:
Great catch!  I missed that earlier.
```suggestion
    TiledCatalogSelector(**optional_args)
```